### PR TITLE
Dev -> Prod

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by HeatherStenson — APPROVED** [review]
Looks good! Redirects seem to be working just fine 🍊

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -212,11 +212,11 @@ The list of acceptable values for `blog_tag` are the slugs for tags in the blog.
 * `workflow`
 * `workspaces`
 
-NOTE: these tags pull from the latest 100 blog posts. We display **up to 3** blog posts for that tag. A return of zero or malformed tag enum will collapse / hide section.
+NOTE: these tags pull from the latest 100 blog posts. We display **up to 3** blog posts for that tag. A return of zero or malformed tag enum will collapse the section.
```
Comment (petewitcher) [inline CONTRIBUTING.md:215]: Should `NOTE: these tags` be `NOTE: These tags`?

**Review by petewitcher — APPROVED** [review]
LGTM! One small question about capitalization.

**Pull request merged**